webui/lib/web/abstract_wait_condition.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABCMeta, abstractmethod
+
+class WaitCondition(object):
```

Comment:
Are we actually going to use all these abstract classes eventually or is it just copypasted from previous project? I just want to be clear on the level of extensibility we're going for.